aldryn_newsblog/views.py

```diff
@@ -188,6 +189,14 @@ class ArticleListBase(AppConfigMixin, AppHookCheckMixin, TemplatePrefixMixin,
     model = Article
     show_header = False
 
+    # Integer that indicates the number after which
+    # we'll start compacting the pagination pages into chunks.
+    pagination_pages_start = getattr(
+        settings, 'ALDRYN_NEWSBLOG_PAGINATION_PAGES_START', 10)
+    # The number of pages visible on each side (left and right).
+    pagination_pages_visible = getattr(
+        settings, 'ALDRYN_NEWSBLOG_PAGINATION_PAGES_VISIBLE', 4)
```

Comment:
@czpython we probably have to add these settings within aldryn as well or? Or can we include them in the namespace settings as we do for "pagination size"?